AnkiDroid/src/main/java/com/ichi2/libanki/Sound.kt

```diff
@@ -84,12 +84,12 @@ class Sound {
     /**
      * Stores sounds for the current card, key is one of the subset flags. It is intended that it not contain empty lists, and code assumes this will be true.
      */
-    private val mSoundPaths: HashMap<SoundSide, ArrayList<String>>? = HashMap()
+    private val mSoundPaths: HashMap<SoundSide, ArrayList<String>?> = HashMap()
```

Comment:
The HashMap property was nullable not the type parameter ArrayList<String>. So it should be:

```suggestion
    private val mSoundPaths: HashMap<SoundSide, ArrayList<String>> = HashMap()
```